package.json

```diff
@@ -39,10 +39,10 @@
   "dependencies": {
     "@openeth/truffle-typings": "0.0.6",
     "@openzeppelin/contracts": "^3.2.0",
-    "@rsksmart/rif-relay-client": "https://github.com/infuy/rif-relay-client#PP-94/revenue-sharing-model",
-    "@rsksmart/rif-relay-common": "https://github.com/infuy/rif-relay-common#PP-94/revenue-sharing-model",
-    "@rsksmart/rif-relay-contracts": "https://github.com/anarancio/rif-relay-contracts#PP-94/revenue-sharing-model",
-    "@rsksmart/rif-relay-server": "https://github.com/infuy/rif-relay-server#PP-94/revenue-sharing-model",
+    "@rsksmart/rif-relay-client": "https://github.com/antomor/rif-relay-client#PP-94/mandatory-collector-address",
+    "@rsksmart/rif-relay-common": "https://github.com/antomor/rif-relay-common#PP-94/mandatory-collector-address",
+    "@rsksmart/rif-relay-contracts": "https://github.com/antomor/rif-relay-contracts#PP-94/fix-truffle-typings-patch",
+    "@rsksmart/rif-relay-server": "https://github.com/antomor/rif-relay-server#PP-94/mandatory-collector-address",
```

Comment:
This change isn't required at all after merging the PRs
- [rif-relay-client](https://github.com/infuy/rif-relay-client/pull/11)
- [rif-relay-common](https://github.com/infuy/rif-relay-common/pull/9)
- [rif-relay-contracts](https://github.com/anarancio/rif-relay-contracts/pull/21)